### PR TITLE
Split card headers out, combine Cards

### LINKED ISSUE #991: Update cards to use Card component
As part of form layout updates, common markup for cards was broken into a component. This issue is to ensure the component (`src/components/Card.js`) is used everywhere applicable.

### PR DESCRIPTION
**Pull request by mparke**
closes #991 

combines Cards, but breaks down the different parts ( headers ) into components

### CODE REVIEW

File: src/users/layouts/IndexPage.js [hunk 1]
```diff
@@ -51,26 +51,28 @@ export class IndexPage extends Component {
...
+                      }
+                    />
                   }
-                  iconClass="user-icon"
-                  dispatch={dispatch}
                 >
                   <div className="row">
                     <div className="col-lg-12">
-                      <div className="SecondaryCard-body-label">Email</div>
+                      <div className="Card-body-label">Email</div>
```
Comment (eatonphil) [inline src/users/layouts/IndexPage.js]: Card-bodyLabel?

Comment (mparke) [inline src/users/layouts/IndexPage.js]: Sure, will fix this up

**Comment by eatonphil** [comment]
First, what happened to the SecondaryCard headers?

Before:

screen shot 2017-03-29 at 10 12 26 am

After:

screen shot 2017-03-29 at 10 12 10 am

Also, if we're going to display the images, we should constrain their size.

**Comment by mparke** [comment]
@eatonphil thanks for posting these screencaps, looking at the image stuff now.

Also will fix the headers I broke.

**Comment by mparke** [comment]
screen shot 2017-03-29 at 2 31 01 pm
screen shot 2017-03-29 at 2 31 12 pm
screen shot 2017-03-29 at 2 31 22 pm

**Comment by mparke** [comment]
@eatonphil I didn't mess with client image response, do we have an issue to address that?

**Comment by eatonphil** [comment]
Secondary cards look good, but we've got some extra padding on normal cards now.

Before:

screen shot 2017-03-29 at 2 40 11 pm

After:

screen shot 2017-03-29 at 2 40 03 pm

**Comment by mparke** [comment]
s/b fixed now:

screen shot 2017-03-29 at 2 48 24 pm

**Pull request merged**